shared/src/main/scala/org/atnos/eff/Member.scala

```diff
@@ -42,8 +42,8 @@ trait MemberInLower2 extends MemberInLower3 {
   implicit def MemberIn3L[L[_], M[_], R[_]]: MemberIn[L, Fx3[L, M, R]] =
     TaggedMemberIn(1)
 
-  implicit def MemberInAppendL[T[_], L, R](implicit append: MemberIn[T, L]): MemberIn[T, FxAppend[L, R]] =
-    AppendMemberIn(isRight = false, append)
+  implicit def MemberInAppendAnyL[T[_], R]: MemberIn[T, FxAppend[Fx1[T], R]] =
+    AppendMemberIn(isRight = false, TaggedMemberIn(1))
```

Comment:
swap implicit def due to ambiguous 😕

Reply:
This is one place where I am very scared with implicits and `eff`. The whole towering of implicits is very ad-hoc.

Reply:
fixed